### PR TITLE
Poojitha added details!!

### PR DESCRIPTION
**Pull request by PoojithaR**
This Pull Request resolves Issue #19 

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

**Comment by PoojithaR** [comment]
I claim!!

File: Poojitha.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    for (i = 1; i <= n; ++i)
+    {
+        printf("%d, ", t1);
+        nextTerm = t1 + t2;
+        t1 = t2;
+        t2 = nextTerm;
+    }
+    return 0;
+}
```
Comment (rahulsnkr) [inline Poojitha.c]: Add your details in a .txt file!

File: Poojithafibo.c [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    for (i = 1; i <= n; ++i)
+    {
+        printf("%d, ", t1);
+        nextTerm = t1 + t2;
+        t1 = t2;
+        t2 = nextTerm;
+    }
+    return 0;
+}
```
Comment (rahulsnkr) [inline Poojithafibo.c]: The question was to implement Fibonacci Search, not to generate the Fibonacci Series.

**Comment by rahulsnkr** [comment]
Add your details to a .txt file and then submit. Also, the question was to implement Fibonacci Search, not to generate the Fibonacci Series.

**Pull request closed without merging**